src/Components/PostCard/PostCard.js

```diff
@@ -0,0 +1,23 @@
+import CardArticle from '../UI/PostArticle.Styled';
+import CardAddress from '../UI/PostAdresse.Styled';
+import CardUserName from '../UI/PostName.Styled';
+
+export default function PostCard({ name, content, contact, mail, mobile }) {
+	return (
+		<CardArticle>
+			<CardUserName>{name}</CardUserName>
+
+			<p>{content}</p>
+
+			<p>{contact}</p>
+
+			<address>
+				<CardAddress>{mail}</CardAddress>
```

Comment:
I'm not quite sure if you need the `<address>` wrapper here. I'd suggest that you directly use the `<address>` tag in your `CardAddress` styled-component instead of the `<p>` OR that you stick to the `<p>` and remove the `<address>`.

Maybe the MDN article helps to decide: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

Reply:
I have removed address